internal/request/graphql/schema/generate.go

```diff
@@ -1051,9 +1051,10 @@ func (g *Generator) GenerateMutationInputForGQLType(obj *gql.Object) ([]*gql.Fie
 		Description: createDocumentDescription,
 		Type:        obj,
 		Args: gql.FieldConfigArgument{
-			"input":   schemaTypes.NewArgConfig(mutationInput, "Create a "+obj.Name()+" document"),
-			"inputs":  schemaTypes.NewArgConfig(gql.NewList(mutationInput), "Create "+obj.Name()+" documents"),
-			"encrypt": schemaTypes.NewArgConfig(gql.Boolean, encryptArgDescription),
+			"input":         schemaTypes.NewArgConfig(mutationInput, "Create a "+obj.Name()+" document"),
+			"inputs":        schemaTypes.NewArgConfig(gql.NewList(mutationInput), "Create "+obj.Name()+" documents"),
+			"encrypt":       schemaTypes.NewArgConfig(gql.Boolean, encryptArgDescription),
+			"encryptFields": schemaTypes.NewArgConfig(gql.NewList(gql.String), encryptFieldsArgDescription),
```

Comment:
todo: The set of valid params is known at 'compile time' here, so instead of a `[String]` the type of the `encryptFields` input should really be `[Enum]` - this will reduce the ability for users to mess it up, and give them nicer intelisense options with GQL clients and ORMs.

Reply:
brilliant! That's a great idea. Thanks